cirq-google/cirq_google/engine/calibration_to_noise_properties.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Tools for converting Calibrations to NoiseProperties.
+
+Given a Calibration "cal", a user can simulate noise approximating that
+calibration using the following pipeline:
+
+    noise_props = cg.noise_properties_from_calibration(cal)
+    noise_model = cg.NoiseModelFromGoogleNoiseProperties(noise_props)
+    simulator = cirq.Simulator(noise=noise_model)
+    simulator.simulate(circuit)
```

Comment:
Nit: Don't forget >>> to show command lines and put in output lines.

Reply:
Done, but we don't have a set standard for this - these rules are applied inconsistently throughout Cirq, and our docs don't mention them.